src/proxy-animation.js

```diff
@@ -568,9 +568,22 @@ function tickAnimation(timelineTime) {
   if (playState == 'running' || playState == 'finished') {
     const timelineTimeMs = fromCssNumberish(details, timelineTime);
 
-    details.animation.currentTime =
-        (timelineTimeMs - fromCssNumberish(details, this.startTime)) *
-            this.playbackRate;
+    let currentTime =
+      (timelineTimeMs - fromCssNumberish(details, this.startTime)) *
+      this.playbackRate;
+
+    const EPS = 1e-3;
```

Comment:
Defer to a separate PR.  The purpose of suggesting that you try this approach was to highlight the source of the problem for non-inclusive end-points.  I do not believe that this is the best way of accomplishing making the endpoints inclusive as it feels hacky.